README.md

```diff
@@ -6,16 +6,133 @@
 
 # Homebase Lite
 
-[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT) [![Build status](https://badge.buildkite.com/680d303398dd927c109b9c0324e7eacf9c7e2e422210ffb746.svg?branch=master)](https://buildkite.com/serokell/homebase-lite)
 
 Homebase Lite is a Tezos smart contract for a decentralized voting system, based
 on “snapshots” of the blockchain.
 
 The contract specification is available in [`docs/specification.md`](docs/specification.md)
 
-## Build Instructions
+## Prerequisites
 
-Run `make` to build everything.
+### Build tools
+
+If you want to build this software from source you will need
+[Haskell Tool Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/)
+
+### Operating system
+
+Please note that only Linux is officially supported.
+If you want to use `tzbtc-client` on MacOS, you can try [building it from source](#build-instructions) using `stack`.
+
+### Executables
+
+In order to use `homebase-lite` executable or run network tests, you will need to obtain `tezos-client` executable.
+`tezos-client` is used for key storing and operation signing.
```

Comment:
```suggestion
In order to use the `homebase-lite` executable or run network tests, you will need to obtain the `tezos-client` executable, which is used for key storing and operation signing.
```
BTW I propose that we use the word `tool` consistently rather than `executable` and `binary`, but I don't have a particularly strong opinion on this, what do you think?

Reply:
@lierdakil did you miss the question under this suggestion?

Reply:
Uh... no, I've seen it, I think I've switched to "tool" in a few places. However, it does start to sound somewhat weird at some point if we're using "tool" everywhere. I've revised it once more, switched to "tool" where it made sense, and left "executable" where it's specifically about executables, and not nebulous "tools". "Binary" is only left in "binary releases".

Reply:
Oh, my bad, I could have sworn there were some more left.

I see none now :ok_hand: